src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -873,13 +873,12 @@ export default class Aircraft {
         const altitude = data[0];
         let expedite = data[1];
         const airport = window.airportController.airport_get();
-        const radioTrendAltitude = radio_trend('altitude', this.altitude, this.fms.altitudeForCurrentWaypoint());
-        const currentWaypointRadioAltitude = radio_altitude(this.fms.altitudeForCurrentWaypoint());
```

Comment:
after #181 is merged, this type of thing will be handled better.

When I moved this to the top of the method, I was under the assumption that the `alititudeForCurrentWaypoint` would remain the same through the execution of this entire method.  I can see, on line 898, that is not the case.  grrr!

In the future this `runAltitude` method will likely be split up into many smaller methods.